test/integration/features/cmd-config.feature

```diff
@@ -174,10 +174,17 @@ user defined options which changes default behaviour of Minishift.
     | property        | value                                                                                            | expected                                                                                        |
     | iso-url         | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso        | https://github.com/minishift/minishift-b2d-iso/releases/download/v1.1.0/minishift-b2d.iso       |
```

Comment:
Remove the row as it contains b2d reference

Reply:
`b2d` still works with minishift, we should remove this when the corresponding code for b2d is also removed

Reply:
Its been a long time when we last supported b2d, now no b2d iso is tested with minilsift releases. So keeping it here creates an impression that we have full support for end user like we do for minishift-centos.iso. 

AFAIU b2d support code in master repo is dead and can be removed later probably in a new code clean up issue

Reply:
> AFAIU b2d support code in master repo is dead and can be removed later probably in a new code clean up issue

Should be done together then, as long as the code lives in master, we should make sure that it works.

Reply:
Anyway this is PR is only for #2926 code clean up for b2d should be done in a separate issue.

Reply:
> as long as the code lives in master, we should make sure that it works.

@anjannath So you mean b2d is still supported ? AFAIK there is no support for b2d from minishift end

Reply:
It is not supported as in we won't add additional patches to `b2d` as of now. But this test is checking the config option, i think there is value in keeping this test. When we remove `b2d` after deprecating it for one release. Then we should remove this. but yeah let others weigh in..